### PR TITLE
Fix network pool name update issue

### PR DESCRIPTION
**Pull request by RayLiu7**
Fix name update issue of network pool
Now we can update network pool name successfully
image

# ISSUE TYPE
- Bugfix Pull Request

##### RESOURCE NAME
Network Pool

# Checklist:

- [x] I have performed a self-review of my own code to ensure there are no formatting, vetting, linting, or security issues
- [x] I have verified that new and existing unit tests pass locally with my changes
- [x] I have not allowed coverage numbers to degenerate
- [x] I have maintained at least 80% code coverage
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have maintained backward compatibility

# How Has This Been Tested?
Please describe the tests that you ran to verify your changes. Please also list any relevant details for your test configuration

- [x] Unit tests
- [x] Acceptance tests

### CODE REVIEW

**Comment by RayLiu7** [comment]
run e2e test

File: examples/resources/powerscale_networkpool/resource.tf [hunk 1]
```diff
@@ -20,7 +20,8 @@ limitations under the License.
 
 # PowerScale network pool allows you to add network interfaces to network pools to associate address ranges with a node or a group of nodes.
 resource "powerscale_networkpool" "pool_test" {
-  #   Required
+  #   Required (subnet and groupnet cannot be modified once designated)
+  #   Recommend using powerscale_groupnet.groupnet_example.name and powerscale_subnet.subnet_example.name to manage network pool together with grouonet and subnet
```
Comment (NavisJ) [inline examples/resources/powerscale_networkpool/resource.tf:24]: typo here "grouonet and subnet"

**Review by NavisJ — APPROVED** [review]

File: examples/resources/powerscale_networkpool/resource.tf (hunk 1 above)
Comment (RayLiu7) [reply examples/resources/powerscale_networkpool/resource.tf:24]: @NavisJ Good catch!

**Comment by RayLiu7** [comment]
run e2e test

**Review by baoy1 — APPROVED** [review]

**Comment by forrestxia** [comment]
https://github.com/dell/terraform-provider-powerscale/pull/89

**Comment by forrestxia** [comment]
run e2e test

**Pull request merged**